spec/models/king_spec.rb

```diff
@@ -4,4 +4,20 @@
   it 'has type of King' do
     expect(subject.type).to eq 'King'
   end
+
+  describe '#valid_move?' do
+    it 'is a public instance method' do
+      expect(subject).to respond_to :valid_move?
+    end
+
+    it 'returns true when a proposed move is within the movement abilities of the piece' do
+      king = create(:king, position_x: 4, position_y: 4)
```

Comment:
For these you can use a `let` to DRY it up a bit.

`let(:king) { create(:king, position_x: 4, position_y: 4) }`

more info: http://betterspecs.org/#let :smile: